feat!: Update hyphenation tokens (#1914)
Amsterdam/design-system: storybook/src/brand/design-tokens/typography.docs.mdx

| :---------------------------------- | :------------------------------------ |
| `var(--ams-typography-font-family)` | `'Amsterdam Sans', Arial, sans-serif` |

## Hyphenation

> Words that are too long for the end of a line will be split with a hyphen and continue on the next line.

Reviewer: ```suggestion
Words that do not fit on a single line will be split with a hyphen and continue on the next line.
```
